apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -272,6 +327,27 @@ type ResourceIdentifier struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+// ResourcePlacementStatus represents the status of a group of resource placed on a selected cluster
+// An empty ClusterDecision indicates it is not scheduled yet.
```

Comment:
Hi Ryan! A nit here: the comment still mentions ClusterDecision; should it be removed?